examples/set-data-points/AppManifest.json

```diff
@@ -20,21 +20,23 @@
                 },
                 {
                     "name": "databroker",
-                    "image": "ghcr.io/eclipse/kuksa.val/databroker",
-                    "version": "v0.17.0"
+                    "image": "ghcr.io/boschglobal/kuksa.val/databroker",
+                    "version": "0.0.2"
+                },
+                {
+                    "name": "databroker_cli",
```

Comment:
This should not be in the AppManifest.json. Why was this added?

Reply:
Removed as discussed